openlibrary/core/models.py

```diff
@@ -764,6 +764,18 @@ def __repr__(self):
         return "<User: %s>" % repr(self.key)
     __str__ = __repr__
 
+    def render_link(self, cls=None):
+        """
+        Generate an HTML link of this user
+        :param str cls: HTML class to add to the link
+        :rtype: str
+        """
+        extra_attrs = ''
+        if cls:
+            extra_attrs += 'class="%s" ' % cls
+        # Why nofollow?
+        return '<a rel="nofollow" href="%s" %s>%s</a>' % (self.key, extra_attrs, web.net.htmlquote(self.displayname))
```

Comment:
I think it is to tell search engines to not index user specific content? Which seems like a good feature, but I'm not sure how consistently this is implemented.